templates/resources/metal_connection.md.tmpl

```diff
@@ -30,11 +30,11 @@ Use this resource to request the creation an Interconnection asset to connect wi
 
 {{tffile "examples/resources/metal_connection/shared_no_token_metal_to_fabric.tf"}}
 
-### Shared Connection with authorization_code Non-redundant NIMF connection from Equinix Metal to a Cloud Service Provider via Equinix Fabric Port
+### Shared Connection with authorization_code Non-redundant Metal-Fabric Integration connection from Equinix Metal to a Cloud Service Provider via Equinix Fabric Port
 
 {{tffile "examples/resources/metal_connection/shared_nimf_to_csp.tf"}}
```

Comment:
The example file reference here, as well as the one below, should be renamed to remove `_nimf_`